modules/Common/src/EasyAbp.BookingService.Common.Domain/EasyAbp/BookingService/AssetOccupancyProviders/ProviderOccupyingInfoModel.cs

```diff
@@ -9,6 +9,11 @@ public class ProviderOccupyingInfoModel : IOccupyingBaseInfo
     public ProviderOccupyingInfoModel(Asset asset, AssetCategory categoryOfAsset, TimeSpan startingTime,
         TimeSpan duration, DateTime date, int volume)
     {
+        if (volume <= 0)
+        {
+            throw new NegativeOccupyingException(volume);
```

Comment:
Please consider the name `InvalidOccupyingVolumeException` since "Negative" doesn't include the case where `volume == 0`.